### ISSUE TITLE
M_LIMIT_EXCEED error

### ISSUE THREAD

**Issue by erdnaxeli** [matrix-org/matrix-appservice-irc#1144]
**Describe the bug**
I saw some errors in my bridge's logs like this one:
```
POST https://matrix.iiens.net/_matrix/client/r0/join/!zyTXygZcMmguRobGIx%3Aiiens.net (@irc_NickServ:iiens.net) HTTP 429 Error: "{\"errcode\":\"M_LIMIT_EXCEEDED\",\"error\":\"Too Many Requests\",\"retry_after_ms\":3274}"
```

But in my appservice configuration (synapse side) I set rate limited to false.

**To Reproduce**
Not sure

**Expected behavior**
I would expect my bridge to not be rate limited.

**Additional context**
version: 0.20.3
Synapse version: 1.19.0

**Comment by Half-Shot** [comment]
That bug was fixed in Synapse 1.20.0 https://github.com/matrix-org/synapse/issues/8139. Update your homeserver :)

**Comment by erdnaxeli** [comment]
Oups, thank you!
